lte/gateway/c/oai/include/n11_messages_types.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+//-----------------------------------------------------------------------------
+/** @struct itti_n11_create_pdu_session_response_t
+ *  @brief carries the PDU Session Establishment Response from SMF to AMF task
+ */
+
+typedef enum sm_session_fsm_state_e {
+  CREATING_0,
+  CREATE_1,
+  ACTIVE_2,
+  INACTIVE_3,
+  RELEASED_4
```

Comment:
why don't you simplify naming and simply use CREATING, CREATE, ACTIVE, INACTIVE, RELEASED without the numbers in the enum names.

Reply:
fixed